### ISSUE TITLE
Display job stability (health report)

### ISSUE THREAD

**Issue by wolverian** [wolverian/jenky#16]
@assimovt had an idea about this.

**Comment by assimovt** [comment]
Yes, I thought of having some kind of health report, but not with any heavy icons. What if we use smileys near the job name?

:) - All good
:| - 50/50
:( - Needs your attention

**Comment by wolverian** [comment]
I've never used the health report indicators myself, so I don't know how important they are and what their use cases are. In any case, they need to be clearly separate from the build status. I kind of like the weather symbolism in that sense, though I've heard from at least one person that it took him weeks to realize what they meant.
